pkgs/by-name/n8/n8n/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  stdenv,
+  lib,
+  nixosTests,
+  fetchFromGitHub,
+  nodejs,
+  pnpm,
+  jq,
+  moreutils,
+  python3,
+  nodePackages,
+  cacert,
+  xcbuild,
+  libkrb5,
+  libmongocrypt,
+  postgresql,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "n8n";
+  version = "1.45.1";
+
+  src = fetchFromGitHub {
+    owner = "n8n-io";
+    repo = "n8n";
+    rev = "n8n@${finalAttrs.version}";
+    hash = "sha256-4Lxygi8LyBFoudNJYmo6wzswmtuv5uBm48eCuJm9hRw=";
+  };
+
+  # this can be removed when pnpm in nixpkgs is new enough for n8n
+  patchPhase = ''
+    runHook prePatch
+
+    jq '.packageManager = "pnpm@${pnpm.version}"' package.json | sponge package.json
```

Comment:
This should be replaced with `removePackageManagerVersion` after https://github.com/NixOS/nixpkgs/pull/317739 is merged.

Reply:
That commit is no longer part of that PR.